openquake/calculators/postproc/compute_rtgm.py

```diff
@@ -115,7 +110,9 @@ def calc_rtgm_df(hcurves, site, site_idx, oq):
     :returns: pandas dataframe with RTGM and related parameters
     :returns: numpy array of conversion factors to max component
     """
+    IMTS = list(oq.imtls)
```

Comment:
The conventions is to use all caps for constants, so IMTS is a bad name. You can use `imts` or `job_imts` or `orig_imts` or whatever you like more.